fe/fe-core/src/main/java/org/apache/doris/nereids/types/DataType.java

```diff
@@ -587,4 +587,8 @@ public Map<Class<? extends DataType>, Promotion<DataType>> build() {
             return (Map) ImmutableMap.copyOf(promotionMap);
         }
     }
+
+    public double length(double high, double low) {
```

Comment:
This name is easy to confuse with meanings like "The length of this data type"

Reply:
rename to `rangeLength`